CodeReview/src/CodeReviewManager.cpp

```diff
@@ -96,15 +89,16 @@ QList<CommentedNode*> CodeReviewManager::loadReview(QString newVersion)
 	auto store = new FilePersistence::SimpleTextFileStore{"."};
 	auto manager = new Model::TreeManager{};
 	manager->load(store, CODE_REVIEW_COMMENTS_PREFIX+newVersion, false);
-	auto list = manager->root();
+	commentedNodes_ = DCast<CommentedNodeList>(manager->root());
```

Comment:
`loadReview()` return a `QList`. Is this necessary? Can't we simply return the `CommentedNodeList`?